### ISSUE TITLE
Post method is not passing data when form submit.

### ISSUE THREAD

**Issue by raj123** [minbrowser/min#1067]
when form is submitted with POST method in target _blank then it does not pass variables in new url. It works fine in other browsers. This issue does not come with GET method as it passes all variables data in url.

**Comment by raj123** [comment]
am i clear on this?

**Comment by PalmerAL** [comment]
Yeah, I understand what you're saying, but I'm not sure what the reason is. Maybe #140?

**Comment by coder963** [comment]
any update on this?

**Comment by PalmerAL** [comment]
not really, I'm pretty sure this is the same as #140 like I mentioned, but there are other things that need to be fixed for that issue to be solved.
